lib/commands/general.js

```diff
@@ -281,12 +290,12 @@ commands.mobileClickAction = async function mobileClickAction (opts = {}) {
 
 // eslint-disable-next-line require-await,no-unused-vars
 commands.updateSettings = async function updateSettings (settings) {
-  throw new errors.NotYetImplementedError();
+  return await this.espresso.jwproxy.command(`/appium/settings`, 'POST', { settings });
```

Comment:
consider removing unnecessary eslint controls from this method

Reply:
please add settings handler to the main driver class, otherwise it won't work. Check my PRs for mac2 driver for more details.

Reply:
It was working, but still added it as per what I see in uiautomator2-driver